pages/linux/opkg.md

```diff
@@ -26,3 +26,11 @@
 - List all the available packages:
 
 `opkg list`
+
+- Find which package owns a file:
+
+`opkg search {{/path/to/file}}`
```

Comment:
```suggestion
- Find out which package owns a file:

`opkg search {{path/to/file}}`
```

absolute path necessary?

Reply:
Yep. I couldn't get it to work with relative paths.